### PR TITLE
feat: create geocoding function

### LINKED ISSUE #7: Function to take in address and get back coordinates
Use googles apis to do a geocode lookup

### PR DESCRIPTION
**Pull request by ethanzitting**
closes #7

### CODE REVIEW

File: src/config.ts [hunk 1]
```diff
@@ -16,3 +16,4 @@ export const REDIS_HOST = process.env.REDIS_HOST || 'localhost';
 export const REDIS_PORT = parseNumber(process.env.REDIS_PORT) || 6379;
 export const REDIS_PASSWORD = process.env.REDIS_PASSWORD || '';
 export const GOOGLE_RECAPTCHA_SECRET_KEY = process.env.GOOGLE_RECAPTCHA_SECRET_KEY || '';
+export const GOOGLE_GEOCODING_SECRET_KEY = process.env.GOOGLE_GEOCODING_SECRET_KEY || '';
```
Comment (glitchedmob) [inline src/config.ts]: Let's rename this to `GOOGLE_MAPS_API_KEY`

Comment (glitchedmob) [reply src/config.ts]: Also can we get this added to the `.env.example` file?

File: src/utils.ts [hunk 2]
```diff
@@ -7,3 +10,22 @@ export function parseNumber(str?: string): number | undefined {
 export function _throw(msg: string): never {
     throw msg;
 }
+
+export async function getCoordsFromAddress(address: string): Promise<any> {
```
Comment (glitchedmob) [inline src/utils.ts]: We need to write a return type for this function. using `any` is a bad idea.  For now you could probably just add an interface `ICoords` that lives next to this function.

**Review by glitchedmob — CHANGES_REQUESTED** [review]

File: src/utils.ts (hunk 2 above)
Comment (ethanzitting) [reply src/utils.ts:18]: mosdef!

File: src/config.ts (hunk 1 above)
Comment (ethanzitting) [reply src/config.ts]: mosdef!

**Pull request closed without merging**